terrapower/physics/neutronics/dragon/dragonExecutor.py

```diff
@@ -146,20 +143,9 @@ def _collectIONames(self):
 
     def writeInput(self):
```

Comment:
I would like to see writeInput() moved a level higher (to runner probably). I don't think the executor should be responsible for executing AND writing input. It should be limited to file management and execution.

If the runner were responsible for writer management, this module would no longer need to refererence dragonFactory. Limiting the scope of this global would be nice.

Reply:
that's a pretty good point. Let me think on it.